int32 frequency
---
float64 bearing_local
bool valid
